changelog.d/11422.bugfix

```diff
@@ -0,0 +1 @@
+Improve performance of various background updates related to device inboxes.
```

Comment:
```suggestion
Improve performance of various background database schema updates.
```

(it's not *just* the device inbox stuff we're targetting here)

Reply:
Yeah but the idea is to merge this changelog with the ones from the other PRs that sprouted out of https://github.com/matrix-org/synapse/issues/11401

Reply:
update those too :)

Reply:
(ah I've just now seen your comment on #11421)